src/cls/IPM/Storage/SystemRequirements.cls

```diff
@@ -19,16 +19,20 @@ Method VersionStringSet(tValue) As %Status
 
 Property Interoperability As %String(VALUELIST = ",enabled,disabled", XMLPROJECTION = "ATTRIBUTE");
 
+Property Health As %String(VALUELIST = ",true,false", XMLPROJECTION = "ATTRIBUTE");
+
 Method CheckRequirements() As %Status
 {
-	Set tSC1 = ..CheckVersion()
-	Set tSC2 = ..CheckInteroperability()
-	Return $$$ADDSC(tSC1, tSC2)
+	Set tSC = $$$OK
+	Set tSC = $SYSTEM.Status.AppendStatus(tSC,..CheckVersion())
+	Set tSC = $SYSTEM.Status.AppendStatus(tSC,..CheckInteroperability())
+	Set tSC = $SYSTEM.Status.AppendStatus(tSC,..CheckHealth())
+	Return tSC
 }
 
 Method CheckVersion() As %Status
 {
-  Set tSystemVersion = +$System.Version.GetMajor() _ "." _ +$System.Version.GetMinor() ;_ "." _ +$System.Version.GetPoint() _ "+" _ $System.Version.GetBuildNumber()
+  Set tSystemVersion = +$SYSTEM.Version.GetMajor() _ "." _ +$SYSTEM.Version.GetMinor() ;_ "." _ +$System.Version.GetPoint() _ "+" _ $System.Version.GetBuildNumber()
```

Comment:
Nit: Can leave it as $System. Honestly easier to not read all caps

Reply:
Sorry about that. It was changed by viscose auto format-on-save. Will change my local dev settings.